microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -1,27 +1,39 @@
 ---
-title: Microsoft Edge WebView2 API Reference
+title: WebView2 API Reference
 description: API Reference for Microsoft Edge WebView2 SDK.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 08/16/2021
+ms.date: 01/25/2022
 ---
-# Microsoft Edge WebView2 API Reference
+# WebView2 API Reference
 
-The Microsoft Edge WebView2 control enables you to host web content in your application using Chromium-based [Microsoft Edge](https://www.microsoftedgeinsider.com) as the rendering engine.  For more information, navigate to [Overview of Microsoft Edge WebView2](./index.md) and [Get started with WebView2](./get-started/win32.md).
-<!-- linking to a parent node of the TOC ("Get started with WebView2") isn't supported, so the above link goes to the first child article, but with link text of the parent TOC node. -->
+The Microsoft Edge WebView2 control enables you to host web content in your application using Chromium-based Microsoft Edge as the rendering engine.
+
+To download the preview channels of Microsoft Edge (Beta, Dev, and Canary), go to [Microsoft Edge Insider](https://www.microsoftedgeinsider.com).
 
 
 <!-- ====================================================================== -->
 ## Languages and Frameworks
-WebView2 is available for the following languages and frameworks.
-*   [Win32 C++](/microsoft-edge/webview2/reference/win32/index)
-*   .NET
-    *   [Core](/dotnet/api/microsoft.web.webview2.core)
-    *   [WPF](/dotnet/api/microsoft.web.webview2.wpf)
-    *   [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
-*   WinRT
-    *   [Core](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
-    *   [COM Interop](/microsoft-edge/webview2/reference/winrt/interop/index)
+
+WebView2 is available for the following languages and frameworks:
+
+* [Win32 C++](/microsoft-edge/webview2/reference/win32/index)
+
+* .NET
+   * [Core](/dotnet/api/microsoft.web.webview2.core)
+   * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
+   * [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
+
+* WinRT
```

Comment:
What is the relation between WinUI 2, UWP, and esp. WinRT? How are they like the same thing?

Why is the parent list item "WinRT", not "WinUI 2" or "UWP"?  In this h2 section, clarify and explicitly state these "equivalent" terms/platform names. Which API Ref applies to which platform?

Reply:
Closing this Comment b/c this PR isn't to clarify, but is to spot-fix word choice, cleanup tagging, and add landing nav.